src/hb-ot-layout-gsubgpos.hh

```diff
@@ -798,8 +798,8 @@ static inline bool intersects_array (const hb_set_t *glyphs,
 				     const void *intersects_data)
```

Comment:
'intersects' implies that at least one element is shared between the two sets. Since your changing this to require that all elements in values are present in glyphs can you change the name of this function to something like: array_is_subset_of. That will better communicate what is being checked.

Reply:
renamed. Thanks!